acceptancetests/jujucharm.py

```diff
@@ -90,14 +90,15 @@ def local_charm_path(charm, juju_ver, series=None, repository=None,
     else:
         charm_dir = {
             'ubuntu': 'charms',
+            'ubuntu-xenial': 'xenial',
             'win': 'charms-win',
             'centos': 'charms-centos'}
         abs_path = charm
         if repository:
             abs_path = os.path.join(repository, charm)
         elif os.environ.get('JUJU_REPOSITORY'):
             repository = os.path.join(
-                os.environ['JUJU_REPOSITORY'], charm_dir[platform])
+                os.environ['JUJU_REPOSITORY'], charm_dir[platform] if series != 'xenial' else charm_dir['ubuntu-xenial'])
```

Comment:
This branch seems nasty. Is there a cleaner way to do this? Why is it required?

Reply:
So, the mission of this test looks unclear. AFAICS we're trying to check if can we deploy xenial workloads on machines with arm64 architecture.

Possible variants on how to fix it (maybe one of them will look cleaner):
- move used charms from the `xenial` folder to the `ubuntu` folder
- start using charms that are published on the charmhub (`juju-qa-dummy-source`, `juju-qa-dummy-sink`)
- (the cleanest way) discuss the mission of this test and rewrite the proper test in a bash-based environment.